build/build.cs

```diff
@@ -1,106 +1,365 @@
-using System.Reflection;
-using System.Runtime.Versioning;
+using System;
+using System.IO;
+using System.Linq;
 using System.Text.RegularExpressions;
-using IntegrationTests;
-using Npgsql;
-using static System.Globalization.CultureInfo;
-using static Bullseye.Targets;
-using static SimpleExec.Command;
-
-namespace build;
-
-internal class Build
+using System.Threading;
+using Nuke.Common;
+using Nuke.Common.IO;
+using Nuke.Common.ProjectModel;
+using Nuke.Common.Tooling;
+using Nuke.Common.Tools.DotNet;
+using Nuke.Common.Tools.Npm;
+using Serilog;
+using static Nuke.Common.Tools.DotNet.DotNetTasks;
+
+class Build : NukeBuild
 {
-    private static void Main(string[] args)
-    {
-        Target("default", DependsOn("test-core", "test-policy", "test-extensions", "test-http", "commands"));
-
-        Target("restore", () => { Run("dotnet", "restore wolverine.sln"); });
-
-        Target("compile", DependsOn("restore"), () =>
+    /// Support plugins are available for:
+    ///   - JetBrains ReSharper        https://nuke.build/resharper
+    ///   - JetBrains Rider            https://nuke.build/rider
+    ///   - Microsoft VisualStudio     https://nuke.build/visualstudio
+    ///   - Microsoft VSCode           https://nuke.build/vscode
+
+    public static int Main () => Execute<Build>(x => x.Test);
+
+    [Parameter("Configuration to build - Default is 'Debug' (local) or 'Release' (server)")]
+    readonly Configuration Configuration = IsLocalBuild ? Configuration.Debug : Configuration.Release;
+    [Solution] readonly Solution Solution;
```

Comment:
Pass in `GenerateProjects = true` to get strongly typed access to the solution projects

Reply:
Sure, will take a look at it.

Reply:
done